### ISSUE TITLE
Any analytics into active devices or usage rate in field?

### ISSUE THREAD

**Issue by nick92675** [google/physical-web#637]
I'm trying to convince a client and our team to do a project using physical web, but obviously people are skeptical to a new technology.   Are we aware of any data showing number of devices currently in market that support phys web, or that are actively using so I can help benchmark and set expectations?  Or projections of what they might be in 3mo, 6mo, 1yr etc?   Or geographic/demographic breakdown.  (Considering a test in Seattle and Portland markets.)

**Comment by alexboogie** [comment]
I haven't seen any device data parsed by protocol but the best resource in terms of net aggregation I've seen thus far is from : https://www.proxbook.com/

**Comment by nick92675** [comment]
Thanks for the lead @alexboogie - I just pulled their proximity marketing in retail report and the most useful thing I found was this stat:  

"Eddystone is the most recent beacon standard by Google.
Eddystone can trigger URL’s in smartphones as well as
notifications within mobile apps and is supported on both
iOS and Android. As of Chrome 49, which was released in
March 2016, 80% of global smartphones are passively
beacon-enabled. Eddystone-URL eliminates the need for
an app. Therefore, we expect a significant increase in new
use cases and companies supporting this standard. "

If anyone has a paid subscription to App Annie there may be some more detailed info in there.  I only have a free account, which doesn't get super in depth.
